src/web/web.ts

```diff
@@ -25,12 +32,29 @@ class App {
     this.app.use("/assets", express.static(path.join(__dirname, "frontend")));
 
     // Controllers
-    this.app.use('/api', new RoomController().router)
-    this.app.use('/api', new AuthenticationController().router)
-    this.app.use('/api', new UserController().router)
+    this.app.use('/api', new RoomController().router);
+    this.app.use('/api', new AuthenticationController().router);
+    this.app.use('/api', new UserController().router);
     this.app.get("/*", (req, res) => {
       res.render("index");
     });
+
+    this.socket.on('connection', (client: SocketIO.Socket) => {
```

Comment:
better have some kind of other class to handle actual socket messaging rather than putting it all in main (like controller classes for REST Api)

Reply:
added socket controller